presto-docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -0,0 +1,105 @@
+===============
+Pinot Connector
+===============
+
+The Pinot connector allows Presto to query data stored in
+`Apache Pinot™ (Incubating) <https://pinot.apache.org/>`_.
```

Comment:
I realize Pinot is incubating but that will change soon. I would suggest to leave this info out here. Or make sure you follow up with a PR whenever it leaves incubation..